next-sample-app-router/src/app/api/metabase/auth/route.ts

```diff
@@ -8,8 +8,7 @@ if (!process.env.NEXT_PUBLIC_METABASE_INSTANCE_URL) {
 }
 
 const METABASE_JWT_SHARED_SECRET = process.env.METABASE_JWT_SHARED_SECRET;
-const NEXT_PUBLIC_METABASE_INSTANCE_URL =
-  process.env.NEXT_PUBLIC_METABASE_INSTANCE_URL;
+const METABASE_INSTANCE_URL = process.env.METABASE_INSTANCE_URL;
```

Comment:
nice, I didn't knew that we can already drop the prefix in app router 👍🏻

Reply:
For an api route only